Import timings test fails under certain conditions

I can reproduce this reliably by running:

```
pytest -m ''
# FAILED tests/unit/test_polars_import.py::test_polars_import - RuntimeError: measuring import timings failed
```

However, if I run the test in isolation, it works fine:

```
pytest tests/unit/test_polars_import.py::test_polars_import -m ''
# PASSED
```

Not sure what is going on here, but it should ideally be fixed.

--- a/py-polars/tests/docs/test_user_guide.py
+++ b/py-polars/tests/docs/test_user_guide.py
@@ -22,7 +22,7 @@ snippet_paths = [p for p in snippet_paths if "visualization" not in str(p)]
 @pytest.fixture(scope="module")
 def _change_test_dir() -> Iterator[None]:
     """Change path to repo root to accommodate data paths in code snippets."""
-    current_path = Path()
+    current_path = Path().resolve()
     os.chdir(repo_root)
     yield
     os.chdir(current_path)

